css-fonts-5/Overview.bs

```diff
@@ -213,6 +213,11 @@ Relative sizing: the 'font-size-adjust' property</h3>
 			text rendered using the first font in the family list
 			will display differently in older user agents
 			that don't support 'font-size-adjust'.
+
+		<dt><dfn>from-font</dfn>
+		<dd>
+			Computes to the <<number>> corresponding to
+			the <a>aspect value</a> of the [=first available font=].
```

Comment:
What is the actual effect of 'from-font' for the primary font? Maybe, no size adjustment, right? Because m and m' are the same in the equation below.
```
u  =  ( m / m′ ) s
s  =  computed 'font-size-property' value
m  =  metric as specified by the 'font-size-adjust' property
m′ =  metric as specified in the actual font
u  =  adjusted font-size to use
```

Reply:
Yes, exactly. The intent is that `from-font` makes it easy to leave the primary font untouched, and adjust fallbacks appropriately to match it.

Reply:
I see. Thanks for the clarification.